data/global_energy_monitor.py

```diff
@@ -0,0 +1,105 @@
+import pandas as pd
+import numpy as np
+
+solar_data = pd.read_excel("Global-Solar-Power-Tracker-May-2022.xlsx", sheet_name=None)
+solar_data = solar_data['Data']
+wind_data = pd.read_excel("Global-Wind-Power-Tracker-May-2022.xlsx", sheet_name=None)
+wind_data = wind_data['Data']
+gas_data = pd.read_excel("Global-Gas-Plant-Tracker-Aug-2022.xlsx", sheet_name=None)
+coal_data = pd.read_excel("Global-Coal-Plant-Tracker-July-2022.xlsx", sheet_name=None)
+
+"""
+Things to extract and put in database:
+Context: 
+Fraction of energy from Coal per region
+Fraction of energy from Gas per region
+
+Action:
+Proposed Transition To (%):
+- Solar
+- Wind
+- Gas
+
+For Q1-4 data summary -> Also have it do more fine grained (per month?) as well
+
+For quarterly data -> line of region (large region) with number of plants of each type and total generating capacity
+Same for future, proposed number of plants and generating capacity for the next quarters as appropriate
+For region, and (subnational unit, State/Province, as they are the same it seems), and Country
```

Comment:
That's what I was going for, I thought GEM had the dates down to the month, but it seems it might only be yearly. They update the gas and coal plant trackers every 6 months, and we can get older ones by somewhat guessing the URLs for them, and diff those. The solar and wind datasets are only updated annually though. I've asked the contact person for solar if they do happen to have more time granularity, or if they know where we could get it.